tests/test_fetcher.py

```diff
@@ -364,9 +330,15 @@ def initialize(context):
 
 def handle_data(context, data):
     if {should_have_data}:
-        assert 'cpi' in data['urban']
+        try:
```

Comment:
Although this isn't behaviour we want to really support, I added this workaround here because it seems like we still want to test this underlying behaviour of fetcher here, despite the changes in the data API.